src/components/common/Button.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styled from "styled-components";
+import { COLORS } from "constants/color";
+import { textAlign, buttonType } from "types";
+
+interface ButtonComponent {
+  children?: React.ReactNode;
+  onClick?: () => void;
+  disabled?: boolean;
+  type?: buttonType;
+  textAlign?: textAlign;
```

Comment:
여기도 마찬가지로 상속받으면 타이핑할게 줄어들겠어요!
```suggestion
interface ButtonComponent extends React.ButtonHTMLAttributes<HTMLButtonElement>{
  children?: React.ReactNode;
  disabled?: boolean;
  textAlign?: textAlign;
```

Reply:
간단하게 할 수 있었는데 괜히 다중 상속 찾고 있었네요..ㅋㅋㅋㅋ

- [refactor: Button 인터페이스 중 네이티브 타입을 상속으로 변경](https://github.com/usageness/react-payments/pull/1/commits/46a7ed441773a9e8a886d6dcc59221a184321ab1)